commands/credentials.js

```diff
@@ -9,11 +9,25 @@ function * run (context, heroku) {
   const {app, args, flags} = context
 
   let showCredentials = co.wrap(function * () {
-    let db = yield fetcher.database(app, args.database)
-    cli.log(`Connection info string:
+    const host = require('../lib/host')
+    let addon = yield fetcher.addon(app, args.database)
+    try {
+      let credentials = yield heroku.get(`/postgres/v0/databases/${addon.name}/credentials`,
+                                       { host: host(addon) })
+      cli.table(credentials, {
+        columns: [
+          {key: 'name', label: 'Credential'},
+          {key: 'state', label: 'State'}
+        ]
+      })
+    } catch (err) {
```

Comment:
that's right, we will be deprecating it in favour of `pg:credentials:rotate --all` in the future; there are plans to add a deprecation warning, communicate with customers who we know are using it in automated scripts: this is still early days though